content/news/release-20191117.md

```diff
@@ -0,0 +1,16 @@
+Just in time for the holiday season, OpenRA release-20191117 is now available with all of the features and fixes that have been tested and refined in the playtest builds over the past few months.
+
+Big new features like save-game support, new UI for Tiberian Dawn and spectators, and completely overhauled unit behaviours make this one of our most exciting releases to date!
+
+For more details, we hand over to FiveAces who has put togther a comprehensive video introduction:
+
+<div style="text-align:center" markdown="1">
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/93b7TKooj-M" frameborder="0" allowfullscreen></iframe>
+
+</div>
+
+
+If you don't like videos, the main changes have been outlined in previous news posts [[1](http://www.openra.net/news/preview-20190606/), [2](http://www.openra.net/news/playtest-20190825/), [3](http://www.openra.net/news/playtest-20191021/)], and, as always, in the  [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/5c17f21e10224a4c31c74fa619ccd22f31539a96). A new [Mod SDK](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20191117) release is also available for the modding community to take advantage of the new release.
```

Comment:
Small nit: [1, 2, 3] is not very good for touch devices. `<a>here</a>, <a>here</a>, and <a>here</a>` is more accessible.